vector/src/main/java/im/vector/notifications/RoomsNotifications.java

```diff
@@ -350,6 +350,25 @@ private void initWearableMessage(Context context, Room room, Event latestEvent,
         }
     }
 
+    /**
+     * Return the total number of unread messages
+     *
+     * @return The total number of unread messages
+     */
+    public int getTotalMessages() {
+        if (mNotifiedEventsByRoomId == null) {
+            return 0;
+        }
+
+        int result = 0;
+
+        for (List list : mNotifiedEventsByRoomId.values()) {
+            result += list.size();
+        }
+
+        return result;
```

Comment:
I would be interested to display only the number of rooms with unread notifications instead of the number of all the notifications

Reply:
Done